docs/quickstart.md

```diff
@@ -53,6 +53,131 @@ qc_mapped_level = get_benchmark(
     level=BenchmarkLevel.MAPPED,
     circuit_size=5,
     target=get_device("ionq_forte_36"),
+    opt_level=2,
 )
 qc_mapped_level.draw(output="mpl")
 ```
+
+## Mirror Circuits
+
+Mirror circuits replicate a given benchmark circuit and its mirror image (inverse) concatenated together. This is useful for performance benchmarking because the ideal final state is known (all qubits should return to their initial state), making deviations easy to detect.
+
+```{code-cell} ipython3
+qc_mirrored = get_benchmark(
+    benchmark="dj",
+    level=BenchmarkLevel.MAPPED,
+    circuit_size=5,
+    target=get_device("ionq_forte_36"),
+    opt_level=2,
+    # generate_mirror_circuit=True,
+)
+qc_mirrored.draw(output="mpl")
+```
+
+## Self-defined Circuits
+
+Sometimes you want to benchmark a circuit that you designed yourself. You can create a `qiskit.QuantumCircuit`, pass it directly to `get_benchmark`, and let `mqt.bench` compile it for your chosen target backend.
+
+```{code-cell} ipython3
+from qiskit import QuantumCircuit
+
+qc = QuantumCircuit(2)
+qc.h(0)
+qc.cx(0, 1)
+
+qc_circuit = get_benchmark(benchmark=qc, level=BenchmarkLevel.NATIVEGATES, target=get_target_for_gateset("ionq_forte", 2))
+qc_circuit.draw(output="mpl")
+```
+
+## Self-defined Targets
+
+You can also provide a target manually. Here we build a 5‑qubit `GenericBackendV2` with a custom basis and attach a short description before feeding it into `get_benchmark`.
+
+```{code-cell} ipython3
+from qiskit.providers.fake_provider import GenericBackendV2
+
+standard_gates = ["id", "x", "sx", "rz", "cx"]
+backend = GenericBackendV2(num_qubits=5, basis_gates=standard_gates)
+target = backend.target
+target.description = "Awesome Target"
+
+qc_target = get_benchmark(
+    benchmark="dj",
+    level=BenchmarkLevel.NATIVEGATES,
+    circuit_size=5,
+    target=target,
+    opt_level=2,
+)
+qc_target.draw(output="mpl")
+```
+
+## Random and Symbolic Parameters
+
+Benchmarks such as QAOA require sets of real parameters. `mqt.bench can either choose random numeric values for you (default) or leave the parameters symbolic so that you can optimize them later.
```

Comment:
```suggestion
Variational benchmarks such as QAOA require sets of real parameters. MQT Bench can either choose random numeric values for you (default) or leave the parameters symbolic so that you can optimize them later.
```